optashift-employee-rostering-gwtui/src/main/java/java/time/ZoneId.java

```diff
@@ -0,0 +1,17 @@
+package java.time;
+
+import java.time.zone.ZoneRules;
+
+// Stub class since the java compiler do not know about supersources
+// See (supersources for REAL class)
+public class ZoneId {
```

Comment:
We can't have a class in the package java.time in src/main/java
Let's talk about why this one is needed.